Docker install doc updates (#19783)
gravitational/teleport: docs/pages/includes/enterprise/docker-images.mdx

Nightly images point to the latest version of Teleport Enterprise from the three most recent release branches.
They are stable, and we recommend their use to easily keep your Teleport Enterprise installation up to date.

> These images are hosted on Amazon ECR Public. All tags under `public.ecr.aws/gravitational/teleport-ent` [are Teleport Enterprise images](https://gallery.ecr.aws/gravitational/teleport-ent).

Reviewer: ```suggestion
These images are hosted on our [Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent). All tags under `public.ecr.aws/gravitational/teleport-ent` are Teleport Enterprise images.
```

To make it clearer that the link is to our image repository

> Versions without a architecture postfix type are `amd64` architecute. (public.ecr.aws/gravitational/teleport:(=teleport.major_version=)). All architecture versions are available 

Reviewer: ```suggestion
Versions without an architecture postfix type use the `amd64` architecture. (public.ecr.aws/gravitational/teleport:(=teleport.major_version=)). All architecture versions are available 
```

> Versions without a architecture postfix type are `amd64` architecute. (public.ecr.aws/gravitational/teleport:(=teleport.major_version=)). All architecture versions are available 

Reviewer: What is "(public.ecr.aws/gravitational/teleport:(=teleport.major_version=))" referring to? Is this an example of an image tag that uses amd64?

Author: Yes, that's a default image.  All are amd64 that don't have suffixes.

> with the postfix type (ex: `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)-arm64`).

Reviewer: I think we can make "All architecture versions are available with the postfix type" clearer. Is this saying that you can pull an image with the `amd64`, `arm`, or `arm64` architecture by specifying it as a suffix?

Author: Yes, that's correct.  I can change to suffix instead of postfix.

> Versions without an architecture postfix type use the `amd64` architecture. (public.ecr.aws/gravitational/teleport:(=teleport.major_version=)). All architecture versions are available 
> with the postfix type (ex: `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)-arm64`).

Reviewer: ```suggestion
You can specify the architecture of a Teleport Docker image by adding a postfix to the image tag, e.g., `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)-arm64`. Versions without an architecture postfix use the `amd64` architecture, e.g., `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)`. 
```
To make the postfix behavior more explicit
